.travis.yml

```diff
@@ -1,14 +1,18 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
+
+os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
     - CONDA_PY=27
     - CONDA_PY=34
     - CONDA_PY=35
+    - CONDA_PY=36
```

Comment:
Seems the development version of Python 3.6 leaked into `main`. This happened with some `broken` labeled packages too.

Have made sure these are in `prerelease` only now. Something to keep an eye on though. Going to close out all these re-renderings that are using 3.6.

cc @jjhelmus @conda-forge/core